.github/workflows/test_models.yml

```diff
@@ -49,6 +49,11 @@ jobs:
           pip install --no-compile --pre --upgrade -r requirements.txt
           pip install --no-compile --pre -e .[testing]
           pip install --no-compile --pre --upgrade -e ${{ github.workspace }}/SHARK-Turbine/models -r ${{ github.workspace }}/SHARK-Turbine/models/requirements.txt
+          pip install \
+            --find-links https://iree.dev/pip-release-links.html \
```

Comment:
This could create some backwards compatibility issues if we are using the latest tree-compiler & runtime, but since you have verified this we can go with this for now.

Reply:
Yeah, since it is just for model testing should be fine for now. But, I can pin it to a specific version here if we ever run into problems. Also, thought it would just be good for visibility to see when our models are not passing using the latest iree.